JP Translation for docs/oncall/alerting_principles.md  v1

JakeMemsource opened the pull request:
Translation is loaded as a first rev. Please check. 
docs/oncall/alerting_principles.md

JakeMemsource committed:
File alerting_principles.md committed.

JakeMemsource committed:
Update1 alerting_principles.md

Updated on Git v1 after pulled from TMS

JakeMemsource committed:
Update v3 alerting_principles.md

Fixed on **Low** priority　

JakeMemsource committed:
Update v4 alerting_principles.md

Spacing on **Low** priority

pd-mhori commented on docs/oncall/alerting_principles.md:
```diff
@@ -1,74 +1,71 @@
 
-!!! warning "High Priority Alerts"
-    Anything that wakes up a human in the middle of the night should be **immediately human actionable**. If it is none of those things, then we need to adjust the alert to not page at those times.
+!!! warning "High Priority Alerts"夜中に人間を起こすものは**即座に人間が実行可能な**でなければなりません。そのようなものがない場合は、その時点でページが表示されないようにアラートを調整する必要があります。
```
ここでの `page` は「通知する」意味合いで使われていそうですね。

```suggestion
!!! warning "優先度の高いアラート"
    夜中に人間を起こすものは**即座に人間がアクションを取れる**ようなものでなければなりません。該当しない場合には、その時点で通知が行われないようにアラートを調整する必要があります。
```

pd-mhori commented on docs/oncall/alerting_principles.md:
```diff
@@ -1,74 +1,71 @@
-#### "Production server disk space is filling, expected to be full in 48 hours. Log rotation is insufficient to resolve."
-This would be a **Medium** priority page, requiring human action soon, but not immediately.
+#### "プロダクション・サーバーのディスク領域がいっぱいになり、48時間でいっぱいになると予想されます。ログのローテーションでは解決できません。"
+これは**Medium** priorityとなり、すぐに人間の行動が必要になりますが、直ちに今すぐというほどではありません。
```
前の文の `immediate` と比べると、`soon` はもう少し緩いニュアンスに訳し分けたいですね。

```suggestion
これは**Medium** priorityとなり、いずれ人間の行動が必要になりますが、直ちに今すぐというほどではありません。
```

pd-mhori commented on docs/oncall/alerting_principles.md:
```diff
@@ -1,74 +1,71 @@
 
-#### "Production service is failing for 75% of requests, automation is unable to resolve."_
-This would be a **High** priority page, requiring immediate human action to resolve.
+#### "プロダクションサービスがリクエストの75%で失敗し、自動化が解決できません。"_
```
```suggestion
#### "プロダクションサービスがリクエストの75%で失敗し、自動化では解決できません。"_
```

pd-mhori commented on docs/oncall/alerting_principles.md:
```diff
@@ -1,74 +1,71 @@
-| Medium | High-Priority PagerDuty Alert during **business hours only**. | Requires human action within 24 hours. |
-| Low | Low-Priority PagerDuty Alert 24/7/365. | Requires human action at some point. |
-| Notification | Suppressed PagerDuty Event. | No response required. Informational only. |
+| 高 | PagerDuty Alert 24/7/365. |  **即時の人間の行動**必要. |
```
```suggestion
| 高 | PagerDuty Alert 24/7/365. |  **即時の人間の行動**が必要. |
```

pd-mhori commented on docs/oncall/alerting_principles.md:
```diff
@@ -1,74 +1,71 @@
 
-#### "An SSL certificate is due to expire in one week."
-This would be a **Low** priority page, requiring human action some time soon.
+#### "SSL証明書の有効期限は1週間です。"
```
```suggestion
#### "SSL証明書の有効期限が1週間後に迫っています。"
```

pd-mhori commented on docs/oncall/alerting_principles.md:
```diff
@@ -1,74 +1,71 @@
 
-#### Make sure to include the metric which triggered the alert somewhere in the body.
-  * <span class="icon bad"></span>  Diskspace on a disk is filling.
+#### アラートをトリガーしたメトリクスは、本体のどこかに含まれていることを確認してください。
```
body = 本体 = アラートの内容ということだと思うので、他のところ含めてその方針でsuggest出しますね。

```suggestion
#### アラートをトリガーしたメトリクスを、アラートの内容に含めるようにしましょう。
```

pd-mhori commented on docs/oncall/alerting_principles.md:
```diff
@@ -1,74 +1,71 @@
-#### The body should also include a description of what the actual problem is, and why it's an issue.
-  * <span class="icon bad"></span>  Disk is full.
-  * <span class="icon good"></span> The disk on this host is at 80% capacity. If it becomes too full it could cause system instability as new files will not be able to be created and current files will not be written to.
+#### ボディには、実際の問題が何であるか、そしてなぜ問題なのかについての説明も含めるべきです。
```
```suggestion
#### アラートの内容には、実際どのような問題なのか、そしてなぜ問題なのかについての説明も含めましょう。
```

pd-mhori commented on docs/oncall/alerting_principles.md:
```diff
@@ -1,74 +1,71 @@
-#### Provide clear steps to resolve the problem, or link to a run book. Alerts with neither of these things are useless.
-  * <span class="icon bad"></span>  Fix it by deleting stuff.
-  * <span class="icon good"></span> Follow the run book here for identifying and resolving disk space issues: https://example.com/runbook/disk. Additionally, you should investigate whether log rotation thresholds are sufficient to prevent this happening again, the following run book has the necessary steps: https://example.com/runbook/log-rotate
+#### 問題を解決するための明確な手順を提供するか、ランブックにリンクします。これらのいずれもないアラートは役に立たない。
+  * <span class="icon bad"></span> 何かを削除して修正します。
+  * <span class="icon good"></span> ディスクスペースの問題を特定して解決するには、次のランブックを参照してください。https://example.com/run book/disk.さらに、ログローテーションのしきい値でこの問題が再発しないようにできるかどうかを調べる必要があります。以下の実行ブックには、必要な手順があります。https://example.com/run book/log-rotate
```
ここでの `run book` は一般的な手順書の意味合いで良さそうですかね。そのイメージでsuggest出しますね。

```suggestion
#### 問題を解決するための明確な手順、または手順書へのリンクを提供します。これらのいずれもないアラートは有用ではありません。
  * <span class="icon bad"></span> 何かを削除して問題を解決しましょう。
  * <span class="icon good"></span> ディスクスペースの問題を特定して解決するには、次の手順書を参照してください：https://example.com/runbook/disk . さらに、ログローテーションのしきい値でこの問題が再発しないようにできるかどうかを調べる必要があります。以下の手順書には、必要な手順が記載されています：https://example.com/runbook/log-rotate .
```

pd-mhori commented on docs/oncall/alerting_principles.md:
```diff
@@ -1,74 +1,71 @@
 
-!!! info "Testing is Critical"
-    An untested alert is equivalent to not having an alert at all. You cannot be sure it will alert you when the time comes. Testing that your alerting actually works is critical to proper service health and should be included in any release planning / deployment efforts.
+!!! info "Testing is Critical"テストされていないアラートは、アラートをまったく持たないことと同等です。時間が来たときにそれがまたアラートを出すか確証を持てません。アラートが実際に機能するかどうかをテストすることは、適切なサービスの健全性にとって重要であり、リリースの計画/デプロイメントの取り組みに含める必要があります。
```
`when the time comes` はちょっと意訳入れました。

```suggestion
!!! info "テストは超重要"
    事前にテストされていないアラートは、アラートをまったく持たないことと同等です。いざという時に、アラートが期待通りに届く保証はありません。アラートが実際に機能するかどうかをテストすることは、しかるべきサービスの健全性にとって重要であり、リリースの計画/デプロイメントの取り組みに含める必要があります。
```

pd-mhori commented on docs/oncall/alerting_principles.md:
```diff
@@ -1,74 +1,71 @@
+!!! info "Testing is Critical"テストされていないアラートは、アラートをまったく持たないことと同等です。時間が来たときにそれがまたアラートを出すか確証を持てません。アラートが実際に機能するかどうかをテストすることは、適切なサービスの健全性にとって重要であり、リリースの計画/デプロイメントの取り組みに含める必要があります。
 
-Make sure to test all new and modified alerts. This is usually covered as part of [Failure Friday](https://www.pagerduty.com/blog/failure-friday-at-pagerduty/) for any new service; however, you should manually test them if you need it more quickly. Some things to test:
+新しいアラートと変更されたアラートをすべてテストするようにしてください。これは通常、新しいサービスの[Failure Friday](https://www.pagerduty.com/blog/failure-friday-at-pagerduty/) の一部として扱われますが、より迅速に必要な場合は手動でテストする必要があります。テストすべき事項:
```
```suggestion
新しいアラートと変更されたアラートをすべてテストするようにしてください。これは通常、新しいサービスの[Failure Friday](https://www.pagerduty.com/blog/failure-friday-at-pagerduty/) の一部として扱われますが、より迅速に必要な場合は手動でテストする必要があります。テストすべき事項は以下のとおりです：
```

pd-mhori commented on docs/oncall/alerting_principles.md:
```diff
@@ -1,74 +1,71 @@
-* Test that the threshold is set appropriately. We don't want noisy alerts.
-* Test that you get alerted for the "No Data" condition if applicable. Generally, receiving no data is the same as breaking your threshold.
-* Test that the alert resolves automatically when the metric returns to normal.
+* しきい値が適切に設定されていることを確認します。私たちは必要以上のアラートを望んでいません。
+* "No Data"条件(該当する場合)のアラートを取得するテスト。一般的に、データを受信しないことは、しきい値を破ることと同じです。
+* メトリクスが正常に戻ったときにアラートが自動的に解決するかどうかをテストします。
```
原文は全部 `Test that...` で始まっているので、和文も統一しますね。

```suggestion
* しきい値が適切に設定されていることをテストします。私たちは必要以上のアラートを望んでいません。
* 場合によっては、データがない条件でアラートを受信するテストをします。一般的に、データを受信しないことはしきい値を超えたのと同等です。
* メトリクスが正常に戻ったときにアラートが自動的に解決するかどうかをテストします。
```

pd-mhori reviewed (commented):
@JakeMemsource 
とりあえず気になった箇所へsuggest出しました！

pd-mhori commented on docs/oncall/alerting_principles.md:
```diff
@@ -1,74 +1,71 @@
 ---
-cover: assets/img/covers/alerting_principles.png
-description: We manage how we get alerted based on a simple principle, an alert is something which requires a human to perform an action. Anything else is a notification, which is something that we cannot control, and for which we cannot make any action to affect it. Notifications are useful, but they shouldn't be waking people up under any circumstance.
+cover: assets/img/covers/alerting\_principles.png description:私たちは、単純な原則に基づいてアラートを取得する方法を管理します。アラートは、人間がアクションを実行することを必要とするものです。それ以外は通知であり、それは私たちがコントロールできないものであり、私たちがそれらに影響を与える行動を取ることができないものです。通知は役に立つが、全ての通知が人を叩き起こすべきものではありません。
```
コメントのテストですー

JakeMemsource replied:
解説ありがとうございました！

pd-mhori commented on docs/oncall/alerting_principles.md:
```diff
@@ -1,74 +1,71 @@
+cover: assets/img/covers/alerting\_principles.png description:私たちは、単純な原則に基づいてアラートを取得する方法を管理します。アラートは、人間がアクションを実行することを必要とするものです。それ以外は通知であり、それは私たちがコントロールできないものであり、私たちがそれらに影響を与える行動を取ることができないものです。通知は役に立つが、全ての通知が人を叩き起こすべきものではありません。
 ---
-We manage how we get alerted based on a simple principle. **An alert is something which requires a human to perform an action**. Anything else is a notification, which is something that we cannot control, and for which we cannot make any action to affect it. Notifications are useful, but they shouldn't be waking people up under any circumstance.
+私たちは、単純な原則に基づいて、どのようにアラートを受けるかを管理します。**アラートは、人間がアクション**を実行する必要があるものです。それ以外は通知であり、それは私たちがコントロールできないものであり、私たちがそれらに影響を与える行動を取ることができないものです。通知は役に立つが、全ての通知が人を叩き起こすべきものではありません。
```
```suggestion
私たちは、単純な原則に基づいて、どのようにアラートを受けるかを管理します。**アラートは、人間がアクション**を実行する必要があるものです。それ以外は通知であり、それは私たちがコントロールできないものであり、私たちがそれらに影響を与える行動を取ることができないものです。通知は役に立ちますが、全ての通知が人を叩き起こすべきものではありません。
```

pd-mhori committed:
Update docs/oncall/alerting_principles.md

pd-mhori committed:
Update docs/oncall/alerting_principles.md

JakeMemsource committed:
Update docs/oncall/alerting_principles.md

rev

Co-authored-by: Meiko Hori <mhori@pagerduty.com>

JakeMemsource committed:
Update docs/oncall/alerting_principles.md

Co-authored-by: Meiko Hori <mhori@pagerduty.com>

JakeMemsource committed:
Update docs/oncall/alerting_principles.md

Co-authored-by: Meiko Hori <mhori@pagerduty.com>

JakeMemsource committed:
Update docs/oncall/alerting_principles.md

Co-authored-by: Meiko Hori <mhori@pagerduty.com>

JakeMemsource committed:
Update docs/oncall/alerting_principles.md

Co-authored-by: Meiko Hori <mhori@pagerduty.com>

JakeMemsource committed:
Update docs/oncall/alerting_principles.md

Co-authored-by: Meiko Hori <mhori@pagerduty.com>

JakeMemsource committed:
Update docs/oncall/alerting_principles.md

Co-authored-by: Meiko Hori <mhori@pagerduty.com>

JakeMemsource committed:
Update docs/oncall/alerting_principles.md

Co-authored-by: Meiko Hori <mhori@pagerduty.com>

JakeMemsource committed:
Update docs/oncall/alerting_principles.md

Co-authored-by: Meiko Hori <mhori@pagerduty.com>

JakeMemsource committed:
Update docs/oncall/alerting_principles.md

テストされていないアラートは　->   事前にテストされていないアラートは
補足で入れました。再度Suggest頂いた際の元の状態に戻していただいても結構です。

Co-authored-by: Meiko Hori <mhori@pagerduty.com>
